### PR TITLE
[Flutter-Parent] Update version and build number and have user agent use them

### PR DESCRIPTION
**Pull request by StephenBrough**

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: apps/flutter_parent/test/api/api_prefs_test.dart [hunk 1]
```diff
@@ -14,20 +14,42 @@
...
+            'buildNumber': '10',
+            'packageName': 'com.instructure.parentapp',
+            'version': '2.0.0',
+          };
+        default:
+          assert(false);
+          return null;
+      }
+    });
+  });
```
Comment (CalvinKern) [inline apps/flutter_parent/test/api/api_prefs_test.dart]: Ew, you should use the shorthand that got added:
SharedPreferences.setMockInitialValues({/*map values go here */});

Comment (CalvinKern) [reply apps/flutter_parent/test/api/api_prefs_test.dart]: Oh wait, lol, just realized this is for package info. Nevermind then :)

File: apps/flutter_parent/test/api/api_prefs_test.dart [hunk 2]
```diff
@@ -214,12 +236,13 @@ void main() {
     expect(ApiPrefs.getHeaderMap(token: 'other token')['Authorization'], 'Bearer other token');
   });
 
-  test('getHeaderMap returns a map with the token passed in', () async {
-    SharedPreferences.setMockInitialValues({
-      _debugKey(ApiPrefs.KEY_TOKEN): 'token',
-    });
```
Comment (CalvinKern) [inline apps/flutter_parent/test/api/api_prefs_test.dart]: did this test get deleted?

File: apps/flutter_parent/pubspec.yaml [hunk 3]
```diff
@@ -40,6 +40,7 @@ dependencies:
   get_it: ^3.0.1
   intl: ^0.15.8
   shared_preferences: 0.5.3+4
+  package_info: 0.4.0+10
```
Comment (CalvinKern) [inline apps/flutter_parent/pubspec.yaml]: Could be inserted alphabetically /shrug

File: apps/flutter_parent/test/api/api_prefs_test.dart (hunk 2 above)
Comment (CalvinKern) [reply apps/flutter_parent/test/api/api_prefs_test.dart]: Oh, nope. Looks like I had a duplicate test name. :hides:

File: apps/flutter_parent/pubspec.yaml (hunk 3 above)
Comment (StephenBrough) [reply apps/flutter_parent/pubspec.yaml]: Haha, I should know better by now 🤣

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1011248&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Pull request merged**